src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/config-detail.jelly

```diff
@@ -2,45 +2,40 @@
 <j:jelly xmlns:j="jelly:core" xmlns:c="/lib/credentials"
          xmlns:f="/lib/form"
          xmlns:scm="/jenkins/scm/api/form">
+  <j:choose>
+    <j:when test="${descriptor.apiUriSelectable}">
+    <f:entry title="${%API endpoint}" field="apiUri">
+      <f:select/>
+    </f:entry>
+    </j:when>
+    <j:otherwise>
+      <f:invisibleEntry>
+        <f:select field="apiUri"/>
+      </f:invisibleEntry>
+    </j:otherwise>
+  </j:choose>
   <f:entry title="${%Credentials}" field="credentialsId">
     <c:select checkMethod="post"/>
   </f:entry>
-  <f:radioBlock name="isConfigurableByScan" value="scan" checked="${instance.isConfiguredByRepoScan()}" title="Github Repository Scan" inline="true">
+  <f:radioBlock name="isConfiguredByUrl" value="false" checked="${instance.configuredByUrl}" title="${%Repository Scan}" inline="true">
```

Comment:
I think this should be negative {!...}

Reply:
Yes, you're right, it should be inverted.